### ISSUE TITLE
Question about Moodle Submission (Peer Review)

### ISSUE THREAD

**Issue by FriedmanM** [Media-Ed-Online/intro-web-dev#46]
Is the peer review submission available currently on Moodle for the entire Project-2? Or is it just for the piece we have created thus far (the single page recipe)?

**Comment by JonSwallow** [comment]
@FriedmanM  I don't know, but I wanted to take time to thank you for reminding me to turn the thing in. I swear I forgot all about. Cheers !!

**Comment by FriedmanM** [comment]
@JonSwallow What are you turning in? Just the one page recipe index? Or did you go further?

**Comment by JonSwallow** [comment]
@FriedmanM  yeah nothing just went and edited my earlier comment it is not open to turn anything in.

**Comment by JustineEvansUM** [comment]
@FriedmanM Peer Reviews are at the end of Projects. Project 2 Peer Review will be two weeks from now. Wipe your brow, you've got some time!

**Comment by FriedmanM** [comment]
@JustineEvansUM Thank you! When will Pt 2 be up?

**Comment by JustineEvansUM** [comment]
We always (hopefully, usually) open new topics on Tuesdays!
